interfaces/acados_matlab_octave/check_acados_requirements.m

```diff
@@ -51,27 +53,46 @@ function check_acados_requirements(varargin)
         end
 
         if ismac
-            if ~verLessThan('matlab', '8.5')
-                filename = strcat('casadi-osx-matlabR2015a-v', CasADi_version, '.tar.gz');
-            elseif ~verLessThan('matlab', '8.4')
-                filename = strcat('casadi-osx-matlabR2014b-v', CasADi_version, '.tar.gz');
-            elseif ~verLessThan('matlab', '8.3')
-                filename = strcat('casadi-osx-matlabR2014a-v', CasADi_version, '.tar.gz');
+        CasADi_version = '3.5.5'; % TODO: update automatic download to later CasADi version.
+            if CasADi_version_numeric < 366
```

Comment:
Seems like this change was with 3.6.0 in CasADi, so we should use 360 and something like the flag I described above?
Or we just remove the CasADi < 3.6 case altogether here. It will anyway not be an automatic download if you change the version in the script manually.